setup.py

```diff
@@ -37,7 +37,7 @@
     ],
     "python_requires": ">=3.8,<3.12",
     "install_requires": requirements,
-    "extras_require": {'test': ['graphix>=0.2.8']},
+    "extras_require": {"test": []},
```

Comment:
As compensation for removing it from the requirement.txt, could you please add 'graphix==0.2.8' here?